php/php.project/nbproject/project.xml

```diff
@@ -147,6 +147,15 @@
                         <specification-version>1.20</specification-version>
                     </run-dependency>
                 </dependency>
+                <dependency>
+                    <code-name-base>org.netbeans.modules.editor.lib</code-name-base>
+                    <build-prerequisite/>
+                    <compile-dependency/>
+                    <run-dependency>
+                        <release-version>3</release-version>
+                        <specification-version>4.18</specification-version>
```

Comment:
As discussed in another PR, the specification version of php.project module should be increased too.

Reply:
Yes, will fix. Thanks!